lib/resources/r4/organization.yaml

```diff
@@ -0,0 +1,127 @@
+---
+name: Organization
+field_name_base_url: http://hl7.org/fhir/r4/organization-definitions.html#Organization
+fields:
+  - name: identifier
+    required: 'No'
+    cardinality: 0..*
+    type: Identifier
+    description: Identifies this organization across multiple systems.
+    url: http://hl7.org/fhir/R4/datatypes.html#Identifier
+    action: create
+    example: |
+      {
+        "identifier": [
+          {
+            "system": "http://hl7.org/fhir/sid/us-npi",
+            "value": "FHIR-NPI-10191",
+            "type": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
+                  "code": "NPI"
+                }
+              ]
+            }
+          }
+        ]
+      }
+    note: |
+      <ul>
+        <li> If set, the provided identifier must meet the following conditions:
+          <ul>
+            <li> Must contain the `system`, `value`, and `type` fields.</li>
+            <li> Must contain only an NPI, OID or TAXID system. </li>
+          </ul>
+        </li>
+      </ul>
+    children:
+
+      - name: type
```

Comment:
This rendered in creates. I misled you. We need Identifier for creates, and then this is reads. My bad

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/326/commits/884311d2afad568b4d627e742466974d702778a3